spring-cloud-azure-samples/feature-management-web-sample/src/main/java/com/example/TargetingContextAccessor.java

```diff
@@ -26,7 +26,7 @@
         TargetingContext context = new TargetingContext();
         context.setUserId(requestContext.getParameter("User"));
         ArrayList<String> groups = new ArrayList<String>();
-        groups.add("Ring3");
+        groups.add(requestContext.getParameter("Group"));
```

Comment:
A user can be in multiple groups. The .NET Core example covers that. Not sure if you want to match that.